safe-global/web-core
# fix: Keyboard support

## What it solves

Resolves #1450 

## How this PR fixes it

- Shows hidden buttons when they are focused
- Fixes the Tab Navigation keyboard support
- Removes redundant links on the dashboard
- Disables keyboard support for safe app images that are embedded via iframe
- Only renders advanced details for transactions if its expanded

## How to test it

1. Open the Safe
2. Tab through the dashboard
3. Observe that each clickable element is only focused once
4. Navigate to the assets page
5. Tab through the page
6. Observe the Send buttons are visible on focus
7. Observe the Tab Navigation at the top of the page is usable via keyboard (arrow keys)
8. Navigate to the NFT Page
9. Observe that the Send Button is visible on focus
10. Navigate to the History
11. Tab through a transaction
12. Observe that advanced details are ignored in the tab order if they are not expanded
